app/controllers/application_controller.rb

```diff
@@ -1,4 +1,15 @@
 # frozen_string_literal: true
 
 class ApplicationController < ActionController::Base
+  before_action :authenticate_user!
+
+  private
+
+  def after_sign_in_path_for(_resource)
```

Comment:
引数を捨てる場合は `*` を使うほうが idomatic な気がします。`_` 始まりは単なる命名の習慣で、変数としては依然有効な気がします。

```suggestion
  def after_sign_in_path_for(*)
```

Reply:
> 引数を捨てる場合は * を使うほうが idomatic な気がします

なるほど、知りませんでした。勉強になります。
後学のために、 `*` を使う方が良いとされている記事を調べてみたものの、見当たリませんでした。参考記事など教えてもらえますでしょうか？

Reply:
> Also, note that a bare * can be used to ignore arguments:
> 
> ```ruby
> def ignore_arguments(*)
> end
> ```

https://ruby-doc.org/core-3.0.2/doc/syntax/methods_rdoc.html#label-Array-2FHash+Argument

Reply:
`*`にされるといざ引数を使おうとするタイミングで「で、この引数は何？」ってなるので、僕は`_resource`の方がいいかなーと思います。好みの問題な気がしますね。

Reply:
@cafedomancer 参考記事、ありがとうございます🙏

@JunichiIto コメントありがとうございます！人によって好みの分かれるところなんだな、と理解できました。